File: src/test/resources/corpus/aluno/maior-elemento-no-main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner leitor = new Scanner(System.in);
        int quantidade = leitor.nextInt();
        int[] numeros = new int[quantidade];
        for (int i = 0; i < quantidade; i++) {
            numeros[i] = leitor.nextInt();
        }

        int maior = numeros[0];
        for (int i = 0; i < quantidade; i++) {
            if (numeros[i] > maior) {
                maior = numeros[i];
            }
        }

        System.out.println(maior);
    }
}

File: src/test/resources/corpus/c/aluno/maior-elemento-no-main.c
#include <stdio.h>

#define TAM 50

int main(void) {
    int v[TAM];
    int n, i, maior;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);

    if (n <= 0 || n > TAM) {
        printf("Quantidade fora do permitido\n");
        return 1;
    }

    for (i = 0; i < n; i++) {
        printf("Digite o numero %d: ", i + 1);
        scanf("%d", &v[i]);
    }

    maior = v[0];

    // compara cada elemento com o maior encontrado ate agora
    for (i = 1; i < n; i++) {
        if (v[i] > maior) {
            maior = v[i];
        }
    }

    printf("O maior numero digitado foi %d\n", maior);
    return 0;
}
